tests/test_util.py

```diff
@@ -195,19 +209,31 @@ def set_integer(self, vref, values):
             self.values[v] = values[i]
 
 class Dummy_FMUModelME2(__ForTestingFMUModelME2):
+
+    # -- Test options --
+    
+    # If true, makes use of the real __ForTesting implementation for nominal_continuous_states,
+    # else just returns 1.0 for each.
+    override_nominal_continuous_states = True
+
+    # Values for nominal_continuous_states. 'test_sparse_option()' and more tests will break
+    # if the values are not calculated from get_ode_sizes() as defined at __init__.
+    # REVIEW: Can we fix the tests instead? I don't know how to.
+    _nominal_continuous_states = None
```

Comment:
Please comment if you know how that tests work. Otherwise I'll just leave this as is, since from what I can tell, it's the old tests that does something wrong.

Reply:
Lets leave it.